glide.yaml

```diff
@@ -14,6 +14,9 @@ import:
   version: ^1.1.0
 - package: github.com/mitchellh/go-homedir
   version: b8bc1bf767474819792c23f32d8286a45736f1c6
+- package: github.com/golang/lint
```

Comment:
Should not it be under `testImport` - just for DEV environment

Reply:
I tried to run `go run vendor/github.com/golang/lint/lint.go` and it fails, because child dependencies are not installed

Reply:
Yes, fixed.